src/main/scala/org/constellation/snapshot/SnapshotTrigger.scala

```diff
@@ -61,10 +63,13 @@ class SnapshotTrigger(periodSeconds: Int = 5, unboundedExecutionContext: Executi
               resetNodeState(stateSet) >>
               snapshotProposalGossipService
                 .spread(
-                  SnapshotProposalGossip(
-                    created.hash,
-                    created.height,
-                    SortedMap(created.publicReputation.toSeq: _*)
+                  Signed(
+                    SignHelp.hashSign(created.hash, dao.keyPair),
```

Comment:
Shouldn't we use here `proposal.hash` instead of `created.hash`? As far as I remember we should sign the proposal, not the snapshot itself

Reply:
Yep, your right. I will fix it.